sam-template.yaml

```diff
@@ -44,7 +44,7 @@ Resources:
           Type: SQS
           Properties:
             Queue: !GetAtt SQSScraperQueue.Arn
-            BatchSize: 10
+            BatchSize: 1
```

Comment:
Could you explain why this is?

Reply:
The handlers were recieving events that looked like {records[{...},{...},{...}...{...}],
So the scraper worker handler would have need to run a loop, instead of this line https://github.com/DemocracyClub/LGSF/blob/e5a3b7bb72448b67bf0e56df34cee6a18dc0b591/lgsf/aws_lambda/handlers.py#L14
It would need to be:
```
for record in event["Records"]:
    message = json.loads(record["body"])
    # get and run scraper
```
I wasn't sure how a single event containing multiple scrapers to be run would effect the logging and output. Seemed simplest to make it one at a time for now.

Reply:
Ah, I think I'd misunderstood what it meant. I'd like to be able to run >1 scraper at a time, but on different invocations. That is, we add 300 items to the queue and they are picked of concurrently at, say, 10 at a time.

I assumed that's what this does, but maybe there's some other Lambda concurrent setting somewhere.

You're right that we want each event to only contain a single job (are those the right words for the things??)